plenum/server/node.py

```diff
@@ -3120,41 +3110,48 @@ def three_phase_key_for_txn_seq_no(self, ledger_id, seq_no):
 
     @measure_time(MetricsName.EXECUTE_BATCH_TIME)
     def executeBatch(self, view_no, pp_seq_no: int, pp_time: float,
-                     valid_reqs: List[Request], invalid_reqs: List[Request],
+                     valid_reqs_keys: List, invalid_reqs_keys: List,
                      ledger_id, state_root, txn_root) -> None:
         """
         Execute the REQUEST sent to this Node
 
         :param view_no: the view number (See glossary)
         :param pp_time: the time at which PRE-PREPARE was sent
-        :param valid_reqs: list of valid client REQUESTs
-        :param valid_reqs: list of invalid client REQUESTs
+        :param valid_reqs: list of valid client requests keys
+        :param valid_reqs: list of invalid client requests keys
         """
-        for req in valid_reqs:
-            self.execute_hook(NodeHooks.PRE_REQUEST_COMMIT, request=req,
+        for req_key in valid_reqs_keys:
+            self.execute_hook(NodeHooks.PRE_REQUEST_COMMIT, req_key=req_key,
```

Comment:
It changes the interface of the plugin. Are we sure there are no plugins who reply on this?